lib/mollie/order.rb

```diff
@@ -0,0 +1,147 @@
+module Mollie
+  class Order < Base
+    STATUS_PENDING    = 'pending'.freeze
+    STATUS_AUTHORIZED = 'authorized'.freeze
+    STATUS_PAID       = 'paid'.freeze
+    STATUS_SHIPPING   = 'shipping'.freeze
+    STATUS_EXPIRED    = 'expired'.freeze
+    STATUS_CANCELED   = 'canceled'.freeze
+    STATUS_COMPLETED  = 'completed'.freeze
+    STATUS_REFUNDED   = 'refunded'.freeze
```

Comment:
This status has been removed (this will be reflected in our public documentation this week).

Reply:
👍